recipe/meta.yaml

```diff
@@ -15,7 +15,7 @@ source:
 
 build:
   noarch_python: True
```

Comment:
I don't know if we support that build variable. I think it should be 'noarch: python'